composer.json

```diff
@@ -10,6 +10,8 @@
         "source": "http://github.com/openemr/openemr"
     },
 	"require": {
+        "spipu/html2pdf": "4.5.0",
+        "setasign/fpdi-tcpdf": "1.6.*"
```

Comment:
Another stupid question. How does this know to get specific versions of fpdi and tcpdf from tecnickcom github repo?